
# HG changeset patch
# User Daniel Atallah <datallah@pidgin.im>
# Date 1360879082 0
# Node ID a860609d9c9ee20a64ca5284f7b50cbc57ae5c92
# Parent  75999131f023b369df6d4f5b2001e153b448be13
Fix from David Walser for a crash in the CAP plugin.

diff --git a/pidgin/plugins/cap/cap.c b/pidgin/plugins/cap/cap.c
--- a/pidgin/plugins/cap/cap.c
+++ b/pidgin/plugins/cap/cap.c
@@ -43,7 +43,7 @@
 	int threshold = purple_prefs_get_int("/plugins/gtk/cap/threshold");
 	int min_minute = (current_minute - threshold) % 1440;
 	int max_minute = (current_minute + threshold) % 1440;
-	char *sql, sta_id = NULL;
+	char *sql, *sta_id = NULL;
 	sqlite3_stmt *stmt = NULL;
 	const char *tail = NULL;
 	int rc;

